README.md

```diff
@@ -175,7 +175,7 @@ Below are the primary files in the project directory
 
 [Bonsai]: https://dev.bonsai.xyz/
 [Foundry]: https://getfoundry.sh/
-[Get Docker]: https://docs.docker.com/get-docker/
+[Docker]: https://docs.docker.com/get-docker/
```

Comment:
this is the same link - so the issue was the markdown shorthand or...?

Reply:
That caught me too initially -- it's https://github.com/risc0/risc0-foundry-template/blob/4e8fc10963f308bf86560593554d8aa4e1ce988d/README.md?plain=1#L102 that's being fixed, not any of the lines being changed

Reply:
ahhhh gross. I quite strongly oppose this syntax for that reason & opt to link the first reference in a page and omit latter on rather than link-ify all the things.

Do we not have a CI link checker setup yet? This should be not hard to do if not - example using https://lib.rs/crates/mlc:
- https://github.com/nuke-web3/book/blob/main/.github/workflows/checks.yml
- https://github.com/nuke-web3/book/blob/main/justfile#L134-L140

also relevant
- https://adventures.michaelfbryan.com/posts/linkchecker/
- ⭐ https://github.com/lycheeverse/lychee